content/help/ops/networking/_index.md

```diff
@@ -4,3 +4,44 @@ description: Helps you manage the networking aspects of a running mesh.
+## Key Istio Concepts
+When attempting to understand, monitor or troubleshoot the networking within an Istio deployment it is critical to understand the fundamental Istio concepts starting with the service mesh.  The service mesh is described in [Architecture](/docs/concepts/what-is-istio/#architecture).  As noted in the architecture section Istio has a distinct control plane and a data plane and operationally it will be important to be able to monitor the network state of both.  The service mesh is a fully interconnected set of proxies that are utilized in both the control and data plane to provide the Istio features.
+
+Another key concept to understand is how Istio performs traffic management.  This is described in [Traffic Management Explained](docs/concepts/#traffic-management).  Traffic management allows fine grained control with respect to what external traffic can enter or exit the mesh and how those requests are routed.   The traffic management configuration also dictates how requests between microservices within the mesh is handled.   Full details on how to configure the traffic management is available here: [Traffic Management Configuration](docs/tasks/traffic-management).
+
+TODO - Do we need to more completely describe gateways?  Not properly covered in concepts.
+## Network Layers Beneath the Mesh
+Istio's service mesh runs on top of the networking provided by the infrastructure environment on which the Istio mesh is deployed.  Istio has certain requirements of the networking layer.  This guide will not attempt to provide any operational insight to the networking layer as many options exist.  In the case of Kubernetes a good reference to understand the container networking layer [Kubernetes Cluster Operator](https://kubernetes.io/docs/user-journeys/users/cluster-operator/foundational/).  Istio has the following requirements of the networking infrastructure:
+* The mapping of a service name to workload IP is discoverable by Pilot (this is more a service discovery requirement than a networking requirement)
+* The Pilot discovery process can reach the environment specific API server for service discovery.
+* Service endpoints have L3 reachability to all endpoints for services in the Istio mesh.
+* Any firewall or ACL rules at the infrastructure environment networking layer don't conflict with any of the Istio layer 3-7 traffic management rules
+# Deployment Guidelines
+TODO - This would be about app deployment not Istio deployment,  but is this section needed?  What sort of topics to cover that are networking oriented.  Only obvious thing that comes to mind is appropriate namespace separation between control plane and applications.
```

Comment:
This is where we should highlight any special rules and restrictions on apps deployed with Istio. For example, this kind of thing: https://github.com/istio/istio.github.io/blob/master/content/help/troubleshooting/index.md#route-rules-have-no-effect-on-my-application

Reply:
That is a good idea.  Originally some of the app specific troubleshooting items didn't seem to fit well,  but I can put them here.